Lib/http/cookiejar.py

```diff
@@ -2022,12 +2020,9 @@ def _really_load(self, f, filename, ignore_discard, ignore_expires):
                 filename)
 
         try:
-            while 1:
-                line = f.readline()
+            while (line := f.readline()) != "":
                 rest = {}
```

Comment:
Note that in the old code, `rest` will always be an empty dictionary after the loop ends; however, since `rest` is not used after the loop, it doesn't matter.

Reply:
That's a good point. If it did matter, we could get around it with yet another assignment expression:

```python
while (rest := {}) or (line := f.readline()) != "":
```

It's not the nicest looking piece of logic :smile: